tests/PasswordFieldTest.php

```diff
@@ -68,14 +70,16 @@ public function testPasswordPersistence()
         $this->assertFalse($m->getField('p')->verify('mypass'));
         $this->assertTrue($m->getField('p')->verify('newpass'));
 
+        $table = $this->getProtected($p, 'data')['data'];
```

Comment:
use `$m->export()` instead

Reply:
using `$m->export` directly, gives a "Model is loaded as an entity, it can not be iterated"
I go this way, which is really similar to the previous one :
`$enc = (new Model($m->persistence))->export(['p'])[1]['p'];`

Reply:
this now is changed back as in #74 from @DarkSide666, for me is ok both, but the problem is not here, is in the failing tests which I can't reproduce on my local environment